x/gov/keeper.go

```diff
@@ -264,7 +264,7 @@ func (keeper Keeper) AddDeposit(ctx sdk.Context, proposalID int64, depositerAddr
 	// Add or update deposit object
 	currDeposit, found := keeper.GetDeposit(ctx, proposalID, depositerAddr)
 	if !found {
-		newDeposit := Deposit{depositerAddr, depositAmount}
+		newDeposit := Deposit{depositerAddr, proposalID, depositAmount}
```

Comment:
Why do we need to store a copy of the proposal ID in each deposit?

Reply:
No.  Technically we don't even need to to store the depositerAddr either.

Reply:
OK - https://github.com/cosmos/cosmos-sdk/issues/1419.